internal/kafka/command_link_tasks_list.go

```diff
@@ -94,7 +94,7 @@ func printHumanTaskOuts(cmd *cobra.Command, taskOuts []serializedTaskOut) error
 	for _, taskOut := range taskOuts {
 		errs := make([]string, 0)
 		for _, err := range taskOut.Errors {
-			errs = append(errs, fmt.Sprintf("(%s) Error Message: %s", err.ErrorCode, err.ErrorMessage))
+			errs = append(errs, fmt.Sprintf("(%s) Error Message: \"%s\"", err.ErrorCode, err.ErrorMessage))
```

Comment:
```suggestion
			errs = append(errs, fmt.Sprintf(`(%s) Error: "%s"`, err.ErrorCode, err.ErrorMessage))
```

Reply:
Another idea, since it looks like all these constants end with "ERROR"
```suggestion
			errs = append(errs, fmt.Sprintf(`%s: "%s"`, err.ErrorCode, err.ErrorMessage))
```

Reply:
Yeah I like the last one as the column name is "Errors" so the context is already there. Will update to that.

Reply:
Updated.